FNPlugin/Collectors/SolarWindCollector.cs

```diff
@@ -141,41 +156,63 @@ public override void OnFixedUpdate()
             }
         }
 
```

Comment:
I just realized you might be talking about the function BELOW this comment. Argh, I am really slow today. If that's the case, I agree - it should work most of the time, but after reading some Kopernicus and other system's forum threads, I can already see where it could fail.